File: BubbleSort.java
class BubbleSort{
    public static void BubbleSort(int[] arr){
        int n = arr.length;
        for (int i=0; i<n; i++){
            boolean swap=false;
            for(int j=i+1; j<n; j++){
                if(arr[j]<arr[i]){
                    int temp=arr[i];
                    arr[i]=arr[j];
                    arr[j]=temp;
                    swap=true;
                }
                if (!swap) break;
            }
        }

    }
    public static void main(String[] args){
        int[] arr={64,25,12,22,11};
        BubbleSort(arr);
        for (int num:arr){
            System.out.println(num);
        }
    }
}

File: SortingAlgos/BubbleSort.py
def bubbleSort(arr:list):
    for i in range(len(arr)-1):
        swap=False
        for j in range(i+1,len(arr)):
            if arr[j]<arr[i]:
                arr[i],arr[j]=arr[j],arr[i]
                swap=True
        if not(swap):   
            break
    return arr
print(bubbleSort([64,25,12,22,11]))
        
